app/config/collections.php

```diff
@@ -1870,6 +1870,17 @@
                 'array' => false,
                 'filters' => [],
             ],
+            [
+                '$id' => ID::custom('name'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
```

Comment:
We can probably drop this to something like 255

Reply:
fixed this